test/_community/collections/Posts/index.ts

```diff
@@ -6,6 +6,12 @@ export const postsSlug = 'posts'
 
 export const PostsCollection: CollectionConfig = {
   slug: postsSlug,
+  versions: true,
+  admin: {
+    livePreview: {
```

Comment:
I don't think we should enable live preview here, especially because we can't guarantee they'll have an app running on `http://localhost:3000`. But I also think that this test config is mostly used to reproduce bugs, so this is not needed most of the time. There's a separate live-preview test suite for this anyway.